scripts/HPC_chatbot_preprocessor/parsed_mds/generic/connecting/connecting_paragraph_10.txt

```diff
@@ -0,0 +1,24 @@
+You can exit the connection at anytime by entering:
+$ exit
+logout
+Connection to login.hpc.ugent.be closed.
+ tip "tip: Setting your Language right"
```

Comment:
tip tip? maybe add the word `locale` here? this is not only about language. and one or lines introduction what a locale is